### ISSUE TITLE
How to get rid of this forbidden message?

### ISSUE THREAD

**Issue by chfw** [alibaba/packer-provider#35]
Hi

I am using my super keys to create ecs image and was kicked out by "RiskControl system".

`--> alicloud-ecs: Error creating instance: Aliyun API Error: RequestId: AFB6A213-884C-4571-A544-70F4126384E0 Status Code: 403 Code: Forbidden.RiskControl Message: This operation is forbidden by Aliyun RiskControl system.`

How am I going to proceed next?

thanks

**Comment by chfw** [comment]
Sorry, it's not my fault. Your server in hangzhou has crashed.

`alicloud-ecs: Failed querying vswitch: Aliyun API Error: RequestId:  Status Code: -1 Code: AliyunGoClientFailure Message: Get https://ecs-cn-hangzhou.aliyuncs.com?AccessKeyId=LTAIvlznBMCitjMX&Action=DescribeVSwitches&Format=JSON&RegionId=eu-central-1&SignatureMethod=HMAC-SHA1&SignatureNonce=8ouKZDdf3Ui6vuR8cc0r3rQz2Ly7sh29&SignatureVersion=1.0&Timestamp=2018-04-19T14%3A46%3A54Z&VSwitchId=vsw-gw8ii1omgje72iy33ht2j&Version=2014-05-26&VpcId=vpc-gw87zidewp5r46zfz3xpk&Signature=Jb2BXpuLcHtcZ%2BCjLLXHbhRIyeM%3D: dial tcp: lookup ecs-cn-hangzhou.aliyuncs.com: no such host`

**Comment by chfw** [comment]
I created a job ticket inside alibabacloud and went through a bank identity check. Then this message is gone.

**Comment by john-peterson** [comment]
My preferred card got stuck on the old card and the query to change default just died silently it's one of the only calls that are not exposed in the CLI. 

I got this from the message centre 

> Please Proceed with the Verification of Your Credit / Debit Card

When I click proceed I get

> The credit/debit card below does not need to be verified.

When I click set as preferred the command does silently and I am in mobile don't know if it makes any difference. It's just dog slow should still work the same

I will wait a week before I bother them maybe they will notice the bug anyway. I just need their shared vGPU A10/12 but it is notoriously sold out anyway

They really need to double and triple their shared fractional GPU machines. I have tried to pursuade GCP to do the same but they don't care about small users 

---

all these were sold out or unavailable in Frankfurt and London they have no other locations in Europe  

~~~
iecs.vgn5i-m1.large
"GPUSpec": "Tesla P4*1/8",
"MemorySize": 6.0,

ecs.sgn6i-vws-m2.xlarge
"GPUSpec": "NVIDIA T4/8",
"MemorySize": 10.0,

ecs.vgn6i-m4.xlarge
"GPUSpec": "NVIDIA T4/4",
"MemorySize": 23.0,

ecs.sgn7i-vws-m2.xlarge
"GPUSpec": "NVIDIA A10*1/12",
"MemorySize": 15.5,

ecs.sgn7i-vws-m2s.xlarge
"GPUSpec": "NVIDIA A10*1/12",
"MemorySize": 8.0,
"CpuCoreCount": 4,
